### DISCUSSION TITLE
模型会自动生成多余内容

### DISCUSSION THREAD

**Discussion by hby96** [zai-org/ChatGLM3#549] [Bad Case]
使用chatglm3-6b对论文内容进行问答，模型经常会回答完问题后，然后自己生成问题并回答，图中[round 3]开始全是模型自己生产的：
![YSS46(@MC06O)YHEOD`AV_N](https://github.com/THUDM/ChatGLM3/assets/38247777/83158d0d-848d-4826-8443-cde749bb80f3)
这是为什么？

**Comment by zRzRzRzRzRzRzR** [comment]
#283  中有相同内容，属于badcase
